src/components/layout/header/helpers.ts

```diff
@@ -0,0 +1,48 @@
+export function getLinks({
+  locale,
+  translations,
+}: {
+  locale: string
+  translations: {
```

Comment:
just nitpicking, nedame to nahoru jako interface?

![image](https://user-images.githubusercontent.com/16448321/105616963-31396c80-5ddb-11eb-952d-3e309cd22deb.png)